src/contracts/utils/ERC1155MetadataPrefix.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.7.4;
+
+import "../interfaces/IERC1155Metadata.sol";
+import "./Ownable.sol";
+
+
+contract ERC1155MetadataPrefix is IERC1155Metadata, Ownable {
```

Comment:
What do we use this contract for? For now we don't I guess?

Reply:
It's just a reference implementation used for testing, giving that this doesn't handle any sensitive logic of the system we could use it as a temporal implementation for production too.